lib/systems/platforms.nix

```diff
@@ -482,6 +482,55 @@ rec {
     };
   };
 
+  # MIPS ABI table transcribed from here: https://wiki.debian.org/Multiarch/Tuples
+
+  # can execute on 32bit chip
+  mips-linux-gnu                = { config = "mips-linux-gnu";                gcc = { arch = "mips32r2"; abi = "o32"; float = "hard"; }; };
+  mipsel-linux-gnu              = { config = "mipsel-linux-gnu";              gcc = { arch = "mips32r2"; abi = "o32"; float = "hard"; }; };
+  mipsisa32r6-linux-gnu         = { config = "mipsisa32r6-linux-gnu";         gcc = { arch = "mips32r6"; abi = "o32"; float = "hard"; }; };
+  mipsisa32r6el-linux-gnu       = { config = "mipsisa32r6el-linux-gnu";       gcc = { arch = "mips32r6"; abi = "o32"; float = "hard"; }; };
+
+  # require 64bit chip (for more registers, 64-bit floating point, 64-bit "long long") but use 32bit pointers
+  mips64-linux-gnuabin32        = { config = "mips64-linux-gnuabin32";        gcc = { arch = "mips64r2"; abi = "n32"; float = "hard"; }; };
+  mips64el-linux-gnuabin32      = { config = "mips64el-linux-gnuabin32";      gcc = { arch = "mips64r2"; abi = "n32"; float = "hard"; }; };
+  mipsisa64r6-linux-gnuabin32   = { config = "mipsisa64r6-linux-gnuabin32";   gcc = { arch = "mips64r6"; abi = "n32"; float = "hard"; }; };
+  mipsisa64r6el-linux-gnuabin32 = { config = "mipsisa64r6el-linux-gnuabin32"; gcc = { arch = "mips64r6"; abi = "n32"; float = "hard"; }; };
+
+  # 64bit pointers
+  mips64-linux-gnuabi64         = { config = "mips64-linux-gnuabi64";         gcc = { arch = "mips64r2"; abi =  "64"; float = "hard"; }; };
+  mips64el-linux-gnuabi64       = { config = "mips64el-linux-gnuabi64";       gcc = { arch = "mips64r2"; abi =  "64"; float = "hard"; }; };
+  mipsisa64r6-linux-gnuabi64    = { config = "mipsisa64r6-linux-gnuabi64";    gcc = { arch = "mips64r6"; abi =  "64"; float = "hard"; }; };
+  mipsisa64r6el-linux-gnuabi64  = { config = "mipsisa64r6el-linux-gnuabi64";  gcc = { arch = "mips64r6"; abi =  "64"; float = "hard"; }; };
```

Comment:
`config` here doesn't do anything. You should move these to `lib/systems/examples.nix`

Reply:
Done